frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -49,6 +65,8 @@ export const cleanFunnelParams = (filters: FilterType): FilterType => {
         ...(filters.properties ? { properties: filters.properties } : {}),
         ...(filters.filter_test_accounts ? { filter_test_accounts: filters.filter_test_accounts } : {}),
         ...(filters.funnel_step ? { funnel_step: filters.funnel_step } : {}),
+        ...(filters.funnel_viz_type ? { funnel_viz_type: filters.funnel_viz_type } : {}),
+        ...(filters.funnel_step ? { funnel_to_step: filters.funnel_step } : {}),
```

Comment:
yeah, i'll have to update this from the backend, these are just the params i was given 😄 but definitely.. `funnel_step` and `funnel_to_step`

Reply:
Well, sometimes you need to work with what you got. If that's too much to change, that's also fine, but then I'd add some `// NB! funky names` comment to warn fellow adventurers of dragons ahead.

Reply:
If it helps, on the backend the mixin is actually called `FunnelPersonsStepMixIn`, since the `funnel_step` parameter is only used for querying parameters. But good call, think we should rename the key value itself to reflect that.